contrib/README.md

```diff
@@ -208,6 +208,9 @@ The following build instructions comply with Ubuntu 20.04 and Amazon Linux 2 env
       ```
 # How to run the JDBC regression tests?
 1. Install Maven: https://maven.apache.org/install.html
+    - Navigate to the Downloads section of the website and copy either the tar.gz or zip binary archive from the link column and run `wget paste_link_here`
```

Comment:
Please make sure my suggestions don't change the behavior, but how about:

- Navigate to the `Downloads` section of the Maven website and copy the link address of either the `tar.gz` or `zip` binary archive from the link column and run `wget paste_link_here`. 
- Navigate to the Maven installation instructions by clicking Use > Install, and follow the provided installation instructions.
- Note: make sure `apache-maven-{version}` is in the same location specified in your PATH variable.